Utility/integration-test.sh

```diff
@@ -6,6 +6,9 @@ JSON_FILES=`ls -d Examples/PDK/*.json`
 # Generate Objective-C models
 .build/debug/plank  --output_dir=Examples/Cocoa/Sources/objc/ $JSON_FILES
 
+#Generate flow types for models
+.build/debug/plank --lang flow  --output_dir=Examples/JS/flow/ $JSON_FILES
```

Comment:
is there anything we can run to validate these are correct flow definitions since they are not compiled?